app/helpers/plus-one.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function plusOne(params/*, hash*/) {
```

Comment:
I can understand why this exists, but there's got to be a better way in the future. If you find one, I'd be interested.

One possibility would be to apply the counter using CSS rules: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters?redirectlocale=en-US&redirectslug=CSS%2FCounters